ember-cli-build.js

```diff
@@ -1,21 +1,24 @@
-/*jshint node:true*/
-/* global require, module */
-var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
+/* eslint-env node */
+const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function(defaults) {
-  var app = new EmberAddon(defaults, {
-    // Add options here
+  let app = new EmberAddon(defaults, {
+    nodeModulesToVendor: [
```

Comment:
@oskarrough it includes dragula for tests only cause ember-cli-build.js relates to dummy application. We should move dragula inclusion to the index.js instead

Reply:
here is an example(it uses ember-cli-node-assets instead of ember-cli-node-modules-to-vendor) https://github.com/AnnotationSro/ember-advanced-combobox/blob/master/index.js